public/img/cds.svg

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
why are there so many new additions under `public/img`? shouldn't the `@gmod/jbrowse` package take care of things like this?

Reply:
The @gmod/jbrowse package doesn't have actual npm features and support yet, so all of the files have to been manually inserted in the root in an img folder, source: https://github.com/cmdcolin/jbrowse_in_react_app/ & from the dev's comment on our issue's page "The @gmod/jbrowse-web npm package is non functional right now, but is our work in progress jbrowse 2 release. Docs for jbrowse 2 are starting to come together at http://jbrowse.org/jb2/"

I know this is less than ideal, but until the package is up and running I think we can live with it, yeah?